contrib/terraform/common/common.tf

```diff
@@ -0,0 +1,93 @@
+variable "temp_basedir" {
+  type    = "string"
+  default = "/tmp"
+}
+
+variable "inventory_deploy" {
+  default = true
+}
+
+resource "random_string" "inventory_dir" {
+  length = 10
+  special = false
+}
+
+locals {
+  inventory_file = "${var.temp_basedir}/terraform-${random_string.inventory_dir.result}/hosts.ini"
+}
+
+output "inventory_file" {
+  value = "${local.inventory_file}"
+}
+
+variable "ssh_key_path" {
+  type    = "string"
+  default = "~/.ssh/id_rsa.pub"
+}
+
+variable "ssh_key_name" {
+  type    = "string"
+  default = "metalk8s"
+}
+
+variable "ssh_key_deploy" {
+  default = true
+}
+
+output "ssh_key_path" {
+  value = "${var.ssh_key_path}"
+}
+
+variable "volume_size" {
+  type    = "string"
+  default = 200
```

Comment:
This looks more like an integer than like a string?

Reply:
there is no integer type in terraform